src/test/java/MusicPlayerTest.java

```diff
@@ -0,0 +1,28 @@
+package src.test.java;
+
+import org.testng.annotations.Test;
+import static org.testng.Assert.assertTrue;
+
+public class MusicPlayerTest {
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221402582-7a31e5f4-448d-49c5-8f93-1a7eabf9bb23.png)

![image](https://user-images.githubusercontent.com/104761482/221402603-91e6e00d-fcda-48cd-a049-67907a2d7a7a.png)